frontend/test/metabase/scenarios/dashboard/dashboard_data_permissions.cy.spec.js

```diff
@@ -43,8 +43,8 @@ describe("support > permissions (metabase#8472)", () => {
       .contains("Dropdown")
       .click();
 
-    // Filter the first card by product category
-    selectDashboardFilter(cy.get(".DashCard").first(), "Title");
+    // Filter the first card by User Address
+    selectDashboardFilter(cy.get(".DashCard").first(), "Address");
```

Comment:
Same problem with https://github.com/metabase/metabase/pull/23403/files#r908199201, so I've changed the tests to operate on User.Address since this field were not cached and it displays a search box when we filter on it.